Formula/muttils.rb

```diff
@@ -3,6 +3,9 @@ class Muttils < Formula
   homepage "https://bitbucket.org/blacktrash/muttils/"
   url "https://bitbucket.org/blacktrash/muttils/get/1.3.tar.gz"
   sha256 "c8b456b660461441de8927ccff7e9f444894d6550d0777ed7bd160b8f9caddbf"
+
```

Comment:
remove this blank line since there is no revision. see comment on libhttpserver.